play-media/src/components/Slider/Slider.tsx

```diff
@@ -57,6 +61,7 @@ const Slider: FC<Props> = ({
         className={className}
       >
         {slides}
+        {customPaginationContainer}
```

Comment:
Minor issue, after clicking on one of the pagination bullets the slider stops autoplaying, not sure if it is the desired behavior.

Also, pagination seems to be above the card in the design.

Reply:
Pagination being on the bottom was discussed with Todor and he approved. It is just not possible to have it on top.